Packs/ExpanseV2/README.md

```diff
@@ -1,28 +1,30 @@
-The Cortex Xpanse pack for Cortex XSOAR provides full coverage of the Expanse Expander and Behavior product capabilities to allow SOCs to automate the defense of their company's attack surface. The integrations included in the pack enable fetching and mirroring of Expanse Issues into Cortex XSOAR incidents, and ingestion of indicators (IPs, domains, and certificates) referring to the corporate network perimeter as discovered by Expanse, a Palo Alto Networks company.
+> **Note**: This Pack, is intended for use with Cortex Xpanse Expander v1, for customers utilizing Expander 2.X (i.e. Active ASM) with Cortex XSOAR, please utilize the [Cortex Xpanse](https://cortex.marketplace.pan.dev/marketplace/details/CortexXpanse/) pack.
+
+The Cortex Xpanse pack for Cortex XSOAR provides full coverage of the Cortex Xpanse Expander v1 product and allows SOCs to automate the defense of their company's attack surface. The integrations included in the pack enable fetching and mirroring of Cortex Xpanse Issues into Cortex XSOAR incidents, and ingestion of indicators (IPs, domains, and certificates) referring to the corporate network perimeter as discovered by Cortex Xpanse, a Palo Alto Networks company.
 
 Through a powerful set of playbooks, analysts can correlate the discovered information with data provided from internal security systems (Palo Alto Networks Cortex Data Lake, Prisma Cloud, and Panorama, Active Directory, Splunk SIEM, etc.) to help pinpoint the right owners of assets and automate remediation.
 
 > **Note**: This Pack, as well as its previously named **Expanse v2** Integration, were renamed to **Cortex Xpanse**. All other content items are still named the same.
 
 ##### What does this pack do?
-- Provides the **Cortex Xpanse** integration (for Expanse Expander and Behavior), which allows XSOAR to collect Expanse Issues and bi-directionally mirror them. Several commands are available to search, tag, and update issues and assets in Expanse. The integration also supports the services API.
+- Provides the **Cortex Xpanse** integration (for Cortex Xpanse Expander), which allows XSOAR to collect Xpanse Issues and bi-directionally mirror them. Several commands are available to search, tag, and update issues and assets in Expander. The integration also supports the services API.
```

Comment:
`and update issues and assets in Expander.`? or `and update issues and assets in Xpanse.`?

Reply:
Expander is the correct team here, thanks for double checking.